source/Makefile.mk

```diff
@@ -221,7 +221,7 @@ endif
 endif
 
 HAVE_FFMPEG     = $(shell pkg-config --exists libavcodec libavformat libavutil && echo true)
-HAVE_FLUIDSYNTH = $(shell pkg-config --atleast-version=1.1.7 fluidsynth && pkg-config --max-version=1.9.9 fluidsynth && echo true)
+HAVE_FLUIDSYNTH = $(shell pkg-config --atleast-version=1.1.7 fluidsynth && echo true)
```

Comment:
Looks great, thanks very much!
Sorry my github is somehow screwed up, does not allow me to comment on the conversation tab without closing the issue, so I will make comments here.

Please make this target develop branch, not master. all new things should now go into develop first, then cherry-picked from there
There are some other small things, I will write them on the relevant lines.